tsdb/head_test.go

```diff
@@ -3179,8 +3179,13 @@ func TestChunkSnapshot(t *testing.T) {
 
 	numSeries := 10
 	expSeries := make(map[string][]tsdbutil.Sample)
+	expHist := make(map[string][]tsdbutil.Sample)
+	expFloatHist := make(map[string][]tsdbutil.Sample)
 	expTombstones := make(map[storage.SeriesRef]tombstones.Intervals)
 	expExemplars := make([]ex, 0)
+	// TODO(marctc): use histograms instead of gauge histograms.
```

Comment:
@codesome I had to use `GaugeHistograms` in the meantime as I'm getting this error when using histograms:
```
 	            	    h: (*histogram.Histogram)({
        	            	-    CounterResetHint: (histogram.CounterResetHint) 2,
        	            	+    CounterResetHint: (histogram.CounterResetHint) 0,
        	            	     Schema: (int32) 1,
```

Reply:
Gauge histogram is ok, you can remove this comment. The 2 vs 0 is just the counter reset information. Gauge should make things simpler to test.